docs/installation-docker.rst

```diff
@@ -0,0 +1,81 @@
+=======================
+Installation via Docker
+=======================
+
+Kuma has experimental support for `Docker`_ as an alternative to the
+Vagrant set up.
+
+.. _Docker: https://www.docker.com/
+
+Docker setup
+============
+
+#. Ensure you have cloned the kuma Git repository and it is your current
+   working directory::
+
+        git clone git@github.com:mozilla/kuma.git
+        cd kuma
+
+#. Install the `Docker Toolbox`_. This will install the Docker engine,
+   along with Docker Compose and Docker Machine.
+
+   .. _Docker Toolbox: https://www.docker.com/products/docker-toolbox
+
+#. Create the virtual machine that Docker will run within::
+
+        docker-machine create --driver virtualbox --virtualbox-memory 2048 kuma
+
+#. Configure the environment variables::
+
+        eval "$(docker-machine env kuma)"
+        export COMPOSE_FILE=`pwd`/docker/docker-compose.yml
+        export COMPOSE_PROJECT_NAME=kuma  # String prepended to every container.
```

Comment:
Are you thinking something like `cdenv` (I think that used to be a thing)? These are standard docker-compose env variables. If you run different containers in different shells you don't want these to be global or hardcoded.

However, the `COMPOSE_FILE` defaults to looking in the CWD for a file named `docker-compose.yml`. I can move this file and this env variable is no longer required.

The `COMPOSE_PROJECT_NAME` also defaults to the CWD directory name which would probably be fine in most cases, so this isn't really required either.

With those 2 changes all we need is to set up the docker-machine env variables via `eval "$(docker-machine env kuma)"`.